packages/aws-cdk-lib/aws-eks/test/cluster.test.ts

```diff
@@ -3962,3 +3962,45 @@ describe('cluster', () => {
     });
   });
 });
+
+describe('deletionProtection', () => {
+  test.each([
+    [true, true],
+    [false, false],
+  ])('deletionProtection(%s) should work', (input, expected) => {
+    // GIVEN
+    const { stack } = testFixture();
+
+    // WHEN
+    new eks.Cluster(stack, 'Cluster', {
+      version: CLUSTER_VERSION,
+      deletionProtection: input,
+      kubectlLayer: new KubectlV31Layer(stack, 'KubectlLayer'),
+    });
+
+    // THEN
+    Template.fromStack(stack).hasResourceProperties('Custom::AWSCDK-EKS-Cluster', {
+      Config: {
+        deletionProtection: expected,
+      },
+    });
+  });
```

Comment:
You can describe test conditions more simply.

```suggestion
  test.each([
    true, false
  ])('deletionProtection(%s) should work', (deletionProtection) => {
    // GIVEN
    const { stack } = testFixture();

    // WHEN
    new eks.Cluster(stack, 'Cluster', {
      version: CLUSTER_VERSION,
      deletionProtection,
      kubectlLayer: new KubectlV31Layer(stack, 'KubectlLayer'),
    });

    // THEN
    Template.fromStack(stack).hasResourceProperties('Custom::AWSCDK-EKS-Cluster', {
      Config: {
        deletionProtection,
      },
    });
  });
```

Reply:
I've simplified the test descriptions as you suggested.